frontend/app/mocks/cct-api.server.ts

```diff
@@ -21,10 +21,17 @@ export function getCCTApiMockHandlers() {
     //
     // Handler for GET requests to retrieve pdf
     //
-    http.get('https://api.example.com/cct/letters/:referenceId', async ({ params, request }) => {
+    http.get('https://api.example.com/cctws/OnDemand/api/GetPdfByLetterId?userid=userId&community=community&id=referenceId', async ({ request }) => {
```

Comment:
Remove the search params and validate in the handler. You can use [Query parameter predicate](https://mswjs.io/docs/best-practices/custom-request-predicate#query-parameter-predicate) to add predicate to the handler and ensure the search params are valid too.

```suggestion
    http.get('https://api.example.com/cctws/OnDemand/api/GetPdfByLetterId', async ({ request }) => {
```

Reply:
not sure if this application will reach a point where the `withSearchParams` higher order function would make sense.  I'm not going to add it in at this time, mostly because I don't want to add complexity.  If multiple mocks start needing to validate query params then it would be a good time to add it in I think.